buildspecs/zos_390-64.spec

```diff
@@ -200,7 +200,7 @@
 		<flag id="module_ddr" value="true"/>
 		<flag id="module_gc_modron_eprof" value="true"/>
 		<flag id="module_gptest" value="true"/>
-		<flag id="module_ifa" value="true"/>
+		<flag id="module_ifa" value="false"/>
```

Comment:
To be consistent, these lines should be completely removed. Typically .spec's don't contain flags with `value="false"`

Reply:
Already removed as suggested.